### PR TITLE
W-14981482 Add MFA step to registration renewal (latest)

### PR DESCRIPTION
**Pull request by chris-campbell9**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.6/modules/ROOT/pages/flex-gateway-renew-certificate.adoc [hunk 1]
```diff
@@ -126,6 +126,11 @@ Replace `<your-username>` and `<your-password>` with Anypoint Platform user cred
 +
 Replace `<path-to-registration-file-in-container>` with the path and filename of the existing `registration.yaml` file. You must specify the file in the container file system, which in this case is `/registration/registration.yaml`.
 ====
```
Comment (valkyrie69) [inline gateway/1.6/modules/ROOT/pages/flex-gateway-renew-certificate.adoc:128]: Is there an extra ==== on line 128? When I view this file, I don't see the note at all.

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc [hunk 2]
```diff
@@ -127,6 +127,11 @@ Replace `<your-username>` and `<your-password>` with Anypoint Platform user cred
 +
 Replace `<path-to-registration-file-in-container>` with the path and filename of the existing `registration.yaml` file. You must specify the file in the container file system, which in this case is `/registration/registration.yaml`.
 ====
```
Comment (valkyrie69) [inline gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc:129]: Extra ==== ?

File: gateway/1.6/modules/ROOT/pages/flex-gateway-renew-certificate.adoc (hunk 1 above)
Comment (chris-campbell9) [reply gateway/1.6/modules/ROOT/pages/flex-gateway-renew-certificate.adoc:128]: Hey, that's for the tabs, not the note.

File: gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc (hunk 2 above)
Comment (chris-campbell9) [reply gateway/1.5/modules/ROOT/pages/flex-gateway-renew-certificate.adoc:129]: Hey, that's for the tabs, not the note.

**Review by valkyrie69 — APPROVED** [review]

**Pull request merged**